gpu/people/include/pcl/gpu/people/label_tree.h

```diff
@@ -124,10 +124,10 @@ namespace pcl
       {
         if(sorted[label].size() == 0)
           return 0;
-        for(size_t i = 0; i < sorted[label].size(); i++)
+        for(auto &i : sorted[label])
```

Comment:
I have some issues with the naming of this variable. You're "breaking convention" by using the name `i` on something which is actually a full blown object and not a simple integer.

At this point I have two potential naming suggestions:
1 - Based on the docstring 
> `@param[in] sorted The matrix of all blobs`

  Call it `blob`

2 - Call it `node` since it seems to have children.